src/tests/intg/test_infopipe.py

```diff
@@ -703,7 +703,7 @@ def test_update_member_list_and_get_all(dbus_system_bus,
     # update local cache for group
     try:
         group_iface.UpdateMemberList(group_id)
-    except dbus.exceptions.DBusException as ex:
+    except dbus.exceptions.DBusException:
```

Comment:
Here it might be better to use the exception instead of deleting the variable to print it as a part of the assert message.

Reply:
I think I got what you requested but please review it just in case.